packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts

```diff
@@ -1,13 +1,13 @@
 import * as AT from './actionTypes'
 import { AB_TESTS } from './model'
-import { MempoolVerifyEMailType } from 'data/components/types'
+import { MempoolVerifyEmailType } from 'data/components/types'
 import { RemoteDataType } from 'core/types'
 
 // types
 export type ABTestNameType = AB_TESTS
 
 export type ABTestCmdType = {
-  command: MempoolVerifyEMailType
+  command: MempoolVerifyEmailType
```

Comment:
What does this have to do with the mempool?

Reply:
Nothing, good point removing it 👍